db/migrate/20220204155632_rename_job_report_template.rb

```diff
@@ -0,0 +1,20 @@
+class RenameJobReportTemplate < ActiveRecord::Migration[5.2]
+  TEMPLATE_NAMES = {
+    "Job invocation - report template" => "Job - Invocation Report",
+  }
+
+  def up
+    TEMPLATE_NAMES.each do |from, to|
+      token = SecureRandom.base64(5)
+      ReportTemplate.unscoped.find_by(name: to)&.update_columns(:name => "#{to} Backup #{token}")
+      ReportTemplate.unscoped.find_by(name: from)&.update_columns(:name => to)
```

Comment:
To be honest I wasn't really paying attention what was going on when the migrations were run, but now I have two templates, one with the old name and one with the new one.

![image](https://user-images.githubusercontent.com/7326770/154255015-455b17a7-bf7a-40ad-9686-0a4d83c29c74.png)

Reply:
Must have been caused by me switching branches.